### DISCUSSION TITLE
Drop support for node 14?

### DISCUSSION THREAD

**Discussion by KATT** [trpc/trpc#2838] [RFC / Ideas]
The current LTS of node 16 and node 14 will meet "end-of-life" 2023-04-30.

To me, it makes sense to drop support for node 14. 

<a href="https://github.com/nodejs/release#release-schedule">

<img src="https://user-images.githubusercontent.com/459267/192972700-a72916f5-4ac1-450a-ba20-8d8c54bb9b75.png" />

> https://github.com/nodejs/release#release-schedule
</a>

Any objections?

**Comment by mmkal** [comment]
I think it's pretty reasonable. Although my team are on node14 still because we deploy trpc to AWS lambda which tends to ship native support for new node versions pretty late. They have up to 16 [right now](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html). IMO it's ok for trpc to force us to upgrade when we move to the next major if it makes maintenance easier.
